wunderground_upload.yaml

```diff
@@ -248,10 +268,12 @@ variables:
           ,['dailyrainin', dailyrainin]
           ,['indoortempf', indoortempf]
           ,['indoorhumidity', indoorhumidity]
+          ,['solarradiation', solarradiation]
+          ,['UV', UV]
           ]%}
       {% if i[1] != '' %}
         {% set data.sensors= data.sensors + ['{}={}'.format(i[0], (i[1] if i[0] == 'rtfreq'
-                                              else (i[1] | round(3)) | string))] %}
+                                              else (i[1] | round(3, default="notavailable")) | string))] %}
```

Comment:
Why do we need a `notavailable` default?

Reply:
This blueprint is created after HA 2021.12.0 and does not have this issue.
Can you provide me a sample to check out?